en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td>
+
+API subscriber name.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### SuccessAPIUsageByApplication
+
+Represents successful(2xx response code) usages of an API by an application.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>apiName</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>apiVersion</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API version.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>apiCreatorTenantDomain</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Tenant domain of the API creator.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationId</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationName</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationOwner</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application name owner.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td></td>
+</tr>
+</tbody>
+</table>
+
+### SuccessSummary
+
+Provides successful(2xx response code) usage summary across all the APIs.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#successvalue">SuccessValue</a>]!</td>
+<td>
+
+List containing successful usage values. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### SuccessValue
+
+Represents successful request count(2xx response code) for a single timestamp. Timestamp is calculated according to the
+provided granularity value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
```

Comment:
When you say "used granularity value" it is not clear as to where it is used and by what. Please check whether the following suggestion gives the same meaning.

```suggestion
Represents a timestamp value that is calculated based on the specified granularity value. For example, if the granularity value is `1d`, the timestamp represents the start of the day calculated adhering to the timezone provided in the time filter.
e.g., timeSpan: '2021-06-21T00:00:00.0000000+05:30'
```